flocx_market/objects/bid.py

```diff
@@ -51,3 +51,13 @@ def save(self):
         db_bid = db.bid_update(
             self.marketplace_bid_id, updates)
         return self._from_db_object(self, db_bid)
+
+    @classmethod
+    def get_all_unexpired(cls):
+        unexpired = db.bid_get_all_unexpired()
+        print(type(unexpired[0]))
```

Comment:
I'm guessing this was left in by mistake?

Reply:
yeah